components/tikv_util/src/lib.rs

```diff
@@ -436,16 +436,34 @@ pub fn set_panic_hook(panic_abort: bool, data_dir: &str) {
 
     let data_dir = data_dir.to_string();
     let orig_hook = panic::take_hook();
+
+    let plain_output = if let Ok(_) = env::var("TIKV_PANIC_PLAIN_OUTPUT") {
```

Comment:
I don't think this is useful at all, `slog` is used in all deployment.

Reply:
It is very useful for me when developing. It is not designed for production usage. That is why I hid it behind an environment variable.

Reply:
I suggest not to include the change. On the one hand, everyone has their preferred way to develop, it's impossible to make everyone happy, so keep code simple is preferred instead. On the other hand, the panic hook is executed only in deployment, keeping the same format in both developing and production cases will get more used to real world case.

To parse it, you can run `python -c "print(raw_input().decode('string_escape'))"`, and paste the log as input.

Reply:
> To parse it, you can run python -c "print(raw_input().decode('string_escape'))", and paste the log as input.

This is a painful workflow since the string is bigger than my terminal window.  I don't think I have any particular preference here. I just want to see a readable backtrace, which is a requirement for development.
One way this can be handled in an automated way externally is by having a log parser and then intelligently re-formatting the log. I do this with json log output piped to jq for example. But I am at a loss for how to make this work here since we already have a plain text oriented format.

Reply:
I think I might prefer this setting in a deployment scenario as well, depending on the exact tooling I am using. Some log tools will make it easy to see the multi-line backtrace string, but others will not, and they might happily pass through the log lines that they do not understand.

Reply:
> since the string is bigger than my terminal window

How about `grep backtrace log | python -c "import sys; map(lambda a: sys.stdout.write(a.decode('string_escape')), sys.stdin)"`?

Reply:
okay, that works 👍 

I can add this to the CONTRIBUTING guide.